packages/flutter_tools/lib/src/commands/daemon.dart

```diff
@@ -448,6 +448,7 @@ class AppDomain extends Domain {
     String dillOutputPath,
     bool ipv6 = false,
     String isolateFilter,
+    bool machine = false,
```

Comment:
All the daemon commands are exclusively machine mode, so defaulting to true instead might make more sense here